editor/src/components/canvas/canvas-utils-scene.spec.browser2.tsx

```diff
@@ -198,6 +198,8 @@ describe('moving a scene/rootview on the canvas', () => {
       await dispatchDone
     })
 
+    await wait(5)
```

Comment:
?

Reply:
Ah yes, this fixed a weird issue that arose in this branch where the editor was treating the second mouse move as having been handled. Which as designed appears to be correct.

Reply:
Ah right, can you add a mega short comment here then please so that nobody comes back here and "tidies" it up!

Reply:
good shout, otherwise this might end up being copy-pasted to other places where it's not needed too

Reply:
Fixed in 71215f224.